src/translator/service.h

```diff
@@ -67,23 +67,7 @@ class Service {
   /// of a model.bin.
   /// @param shortlistMemory byte array of shortlist (aligned to 64)
   /// @param vocabMemories vector of vocabulary memories (aligned to 64)
-  explicit Service(Ptr<Options> options, AlignedMemory modelMemory,
-                   AlignedMemory shortlistMemory,
-                   std::vector<std::shared_ptr<AlignedMemory>> vocabMemories);
-
-  /// Construct Service purely from Options. This expects options which
-  /// marian-decoder expects to be set for loading model shortlist and
-  /// vocabularies from files in addition to parameters that set unset desired
-  /// features (e.g: alignments, quality-scores).
-  ///
-  /// This is equivalent to a call to:
-  /// ```cpp
-  ///    Service(options, AlignedMemory(), AlignedMemory(), {})
-  /// ```
-  /// wherein empty memory is passed and internal flow defaults to file-based
-  /// model, shortlist loading. AlignedMemory() corresponds to empty memory
-  explicit Service(Ptr<Options> options)
-      : Service(options, AlignedMemory(), AlignedMemory(), {}) {}
+  explicit Service(Ptr<Options> options, MemoryBundle memoryBundle={});
```

Comment:
Could you please update the documentation of all the 3 constructors of this class?

Reply:
Yes, I will do (trying to fix the WASM build first)

Reply:
@qianqianzhu Please let me know if you need help there 👍